go.mod

```diff
@@ -5,7 +5,7 @@ go 1.23.0
 toolchain go1.24.1
 
 require (
-	github.com/chia-network/go-chia-libs v0.21.5
+	github.com/chia-network/go-chia-libs v0.21.7-0.20250430011307-04bcc3c2b364
```

Comment:
This should get updated to an actual release when https://github.com/Chia-Network/go-chia-libs/pull/203 is merged